use-cases/xtclim/Requirements.txt

```diff
@@ -1,17 +1,15 @@
-cartopy
-cftime
-codecarbon
-dask
-datetime
+cartopy
+cftime
+codecarbon
+dask
+datetime
 imageio
-ipykernel
-matplotlib
-numpy
-pandas
-torch
-torchvision
-tqdm
-urllib3==1.26.13
+ipykernel
+matplotlib
+numpy
+pandas
+tqdm
+urllib3==1.26.13
```

Comment:
I would leave the path version free to change, if possible: `urllib3==1.26.13` -> `urllib3==1.26.*`